test/model/src/core/util.cpp

```diff
@@ -283,6 +283,26 @@ shared_memory::~shared_memory()
     }
 }
 
+/*
+ * parse_uint64 --
+ *     Parse the string into a number. Throw an exception on error.
+ */
+uint64_t
+parse_uint64(const char *str, char **end)
+{
+    if (str == nullptr || str[0] == '\0')
+        throw std::runtime_error("Cannot parse a number");
+
+    char *p = nullptr;
+    uint64_t r = (uint64_t)strtoull(str, &p, 0 /* automatically detect "0x" for hex numbers */);
```

Comment:
You could make it more C++ by:
#include <cstdlib>
Change strtoull() to std:strtoull().

Reply:
Added the `std::` in [70241f1](https://github.com/wiredtiger/wiredtiger/pull/10267/commits/70241f1a1fcf8c5b60f1e9303b46e4568d643a27). I'd like to keep the `parse_uint64`, because I would like it to throw an exception on error.